backend/api/src/main/java/com/virtuslab/gitmachete/backend/api/hooks/ExecutionResult.java

```diff
@@ -0,0 +1,10 @@
+package com.virtuslab.gitmachete.backend.api.hooks;
+
+import lombok.Data;
+
+@Data
```

Comment:
Don't we use java 17 now? Can we use `record` here?

Reply:
I'd rather introduce records in a more coordinated manner, all at once across the entire codebase